### PR TITLE
Created User model and User Model Manager

### PR DESCRIPTION
**Pull request by prashant2796**

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        
+        user.save(using=self._db)
+        return user
+
+
+class User(AbstractBaseUser):
+
+	email = models.EmailField(max_length=255,unique=True)
+	active = models.BooleanField(default=True)
+	staff = models.BooleanField(default=False)  # a admin user; non super-user
```
Comment (darth-dodo) [inline accounts/models.py]: why are we still commenting like this? why do I have to fix same errors/reviews for every pr?!!

File: accounts/models.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+		return self.staff
+
+	@property
+	def is_admin(self):
+		"Is the user a admin member?"
+		return self.admin
+
+	@property
+	def is_active(self):
+		"Is the user active?"
```
Comment (darth-dodo) [inline accounts/models.py]: this is not how docstrings are written

**Comment by darth-dodo** [comment]
https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html

Read up this. Think again about your choices. Explain why you chose this one


What if I want to create three kinds of users in the future, client user, admin user and superuser. How will you schema grow/approach that?

File: accounts/models.py (hunk 2 above)
Comment (prashant2796) [reply accounts/models.py]: ya sorry for this. a big silly mistake went unnoticed.

**Pull request closed without merging**